README.md

```diff
@@ -173,6 +173,8 @@ You may have seen a dialog saying "You are missing the needed firmware files [..
 
 To load firmwares, the easiest way is to click "Import" in the menubar, navigate to the dumped firmware(s), select them all, and click "Open". It's a good idea to have them copied into the `Firmware` folder, which is nicely organised, when prompted. If you were trying to open a rom, click "Close and reload ROM" to do that. Keep in mind some firmware is optional and some have multiple versions, only one of which needs to be set.
 
+Windows users can keep the `Firmware` folder made by the firmware manager when updating, go to `Config` > `Paths...` and change "Firmware" under "Global" to point at it.
```

Comment:
It's missing the main point. _All firmware that hawk needs will be automatically detected_ in whatever folder it uses for Firmware. By default it's Firmware folder in the hawk dir. Throw everything there and you don't have to open firmware manager at all. And if one wants, it can be an external folder all hawks can safely share.

Reply:
FYI: firmware manager (and all related Path functions that were failing before) are now fixed in the working fork under linux.

Reply:
firmware manager *already* worked, it's this simpler method that didn't work

Reply:
fixed btw @vadosnaprimer